src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1232,7 +1255,22 @@
     "useHttp": {
       "type": "computed",
       "dataType": "bool",
-      "value": "(httpEvaluator && useConfiguration)"
+      "value": "(httpEvaluator != \"none\" && useConfiguration)"
+    },
+    "useHttpDefault": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "(httpEvaluator == \"basic\" && useConfiguration)"
```

Comment:
I think these conditions should rely on `useHttp`. So it could be `(usHttp && httpEvaluator == \"basic\")`, etc.

Just so it relies on the base `useHttp` condition if that ever changes to a different condition